package.json

```diff
@@ -192,6 +192,6 @@
     "typescript": "^4.9.3"
   },
   "engines": {
-    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
+    "node": "^12.13.0 || ^14.15.0 || >=16.13.0"
```

Comment:
[ask] It is likely to accept `lint-staged` downgrade because it's for dev, but `engines.node` downgrade may affect users. Is the latter necessary?